### PR TITLE
24 garden show sensors

### LINKED ISSUE #24: As a logged in user, I can view all sensors in a specific garden.

### PR DESCRIPTION
**Pull request by AngelaGuardia**
closes #24 

- Added sensor details to garden show page
- Changed sensor PORO to match data structure from the backend serializer

### CODE REVIEW

File: app/controllers/gardens_controller.rb [hunk 1]
```diff
@@ -7,8 +7,8 @@ def show
     conn = Faraday.new("https://solar-garden-be.herokuapp.com/api/v1/gardens/#{params[:id]}")
     response = conn.get
     parsed = JSON.parse(response.body, symbolize_names: true)
-
     garden = Garden.new(parsed[:data])
+    @sensors = garden.sensors.map { |sensor| Sensor.new(sensor)  }
```
Comment (LHJE) [inline app/controllers/gardens_controller.rb:11]: Clever!

File: app/poros/sensor.rb [hunk 2]
```diff
@@ -4,10 +4,10 @@ class Sensor
               :sensor_type,
               :min_threshold,
               :max_threshold
-              
+
   def initialize(data)
     @id = data[:id]
-    @garden_id = data[:attributes][:garden_id]
+    @garden_id = data[:relationships][:garden][:data][:id]
```
Comment (LHJE) [inline app/poros/sensor.rb:10]: Nice, since now it's under relationships.  Nicely Done!

File: spec/features/gardens/show_spec.rb [hunk 3]
```diff
@@ -1,120 +1,172 @@
...
+
+      visit "/gardens/3"
 
-      expect(page.status_code).to eq(404)
+      within '.garden-sensors' do
+        expect(page).to have_content(@sensor1[:id])
+        expect(page).to have_content(@sensor1[:attributes][:sensor_type])
+        expect(page).to have_content(@sensor2[:id])
+        expect(page).to have_content(@sensor2[:attributes][:sensor_type])
+      end
```
Comment (LHJE) [inline spec/features/gardens/show_spec.rb:169]: Lots of changes, and all of it looks great!

**Review by LHJE — APPROVED** [review]

**Review by dcoleman21 — APPROVED** [review]
Nice job here! That sensor json data looks great! If there is anything that you want us to know ping us.

**Pull request merged**